### PR TITLE
Sync externals

### PR DESCRIPTION
**Pull request by JonathanMeans**

### CODE REVIEW

**Comment by hobu** [comment]
Is there any cycles to get the Mac build happy? I took a look, but I didn't understand what the coda-oss processing stuff was or how it was supposed to be adapted for OSX.

**Comment by JonathanMeans** [comment]
Unfortunately, supporting mac is not a priority for us.

Looking at `externals/coda-oss/modules/c++/mt/include/mt/CPUAffinityInitializer.h`, it looks like 
 - The mac implementation simply doesn't exist, but...
 - I don't think any of the NITRO code actually needs it

A low-effort way of fixing this might be to provide a trivial subclass of `AbstractCPUAffinityInitializer` that just returns `nullptr` for its overload. It looks like the using class `ThreadGroup` is always able to handle a null result. We won't get the expected performance benefit of course, but that doesn't matter for you because NITRO should never call it.

File: externals/coda-oss/modules/drivers/numpy/wscript [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import os, sys
 from os.path import join, exists
+from distutils import sysconfig
```
Comment (vkrishna1997) [inline externals/coda-oss/modules/drivers/numpy/wscript]: Unused import?

**Comment by vkrishna1997** [comment]
Looks good other than that

File: externals/coda-oss/modules/drivers/numpy/wscript (hunk 1 above)
Comment (JonathanMeans) [reply externals/coda-oss/modules/drivers/numpy/wscript]: Resolved in coda-oss PR. I'll merge this and let the cleanup come in whenever it updates next.

**Pull request merged**